### DISCUSSION TITLE
PicviewerCE: which rule has preference?

### DISCUSSION THREAD

**Discussion by Omaeda** [hoothin/UserScripts#772] [Q&A] [answered]
I am trying to change the rule for danbooru, but it seems that the default rule takes precedence over the one I created. Is this really the case or am I doing something wrong?

In the pvcep_rules.js file I have seen that some rules have the option `enabled: true` that means that there must be some place where I can disable them.

I am using Violentmonkey and version 2024.10.27.1 of the script.

**Comment by hoothin** [comment, marked as answer]
You must use the same name (for example, 'name': 'donmai') to overwrite the rules; otherwise, they will coexist, and that might be the issue.

**Reply by miluju** [reply]
保存
